internal/btf/btf.go

```diff
@@ -190,6 +190,22 @@ func loadSpecFromELF(file *internal.SafeELFFile) (*Spec, error) {
 	return spec, nil
 }
 
+// GetFuncLineInfos returns a map of LineInfo's for the given function name. The map is indexes by the
+// instruction offset of the lineInfo.
+func (spec *Spec) GetFuncLineInfos(funcName string) map[uint32]LineInfo {
+	if spec.lineInfos == nil {
+		return make(map[uint32]LineInfo)
```

Comment:
This can just be `nil`, it's allowed to do a lookup in a nil map.

Reply:
:open_mouth: I thought that reading from an nil map also caused a nil pointer panic like writing, learned something today. Will update this